### DISCUSSION TITLE
Crop bug between 2.* and 3.*

### DISCUSSION THREAD

**Discussion by TheROPFather** [spatie/image#259] [General]
Hey, 

I had an existing crop in place using the old 2.* package and since upgrading to 3.* the crop result is different. I am passing the same parameters as far as I can tell, to demonstrate I setup two tiny git repo's so you can easily reproduce. 

Working (2.*) - [Working Spatie Crop](https://github.com/TheROPFather/working-spatie-crop )

Not Working (3.*) - [Broken Spatie Crop](https://github.com/TheROPFather/broken-spatie-crop)

Please clone both, then 
`composer install && php artisan storage:link && cp .env.example .env && php artisan key:generate && php artisan serve` 
the home page will show the original and cropped. The two have different cropped results. 

The code is in the web.php route file.

Any help appreciated, cheers!!

**Comment by ibrainventures** [comment]
Hi i had the same issue.

To get the same result dont use` crop(..., CropPosition: ..) ` !

Use : 

`->fit(Fit::Crop, 614, 853)`

(and imported : use Spatie\Image\Enums\Fit;)
